### ISSUE TITLE
Replace jQuery script form googleapis.com

### ISSUE THREAD

**Issue by manuels** [panzi/SocialSharePrivacy#72]
The [github pages site](https://panzi.github.io/SocialSharePrivacy/) of this project suggests to use the jquery script from Google's CDN googleapis.com

    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

This basically renders this plugin useless with respect to tracing by Google Inc. This link should be replaced with another link (maybe hosted by http://panzi.github.io/)

**Comment by panzi** [comment]
Yes. I'm currently sick in bed, so if you would send me a pull request with appropriate changes that would be nice. 😄
